core/src/main/java/tech/tablesaw/api/LongColumn.java

```diff
@@ -692,8 +692,8 @@ public int byteSize() {
     public LongColumn difference() {
         LongColumn returnValue = new LongColumn(this.name(), data.size());
         returnValue.append(LongColumn.MISSING_VALUE);
-        for (int current = 1; current > data.size(); current++) {
-            returnValue.append(get(current) - get(current + 1));
+        for (int current = 1; current < data.size(); current++) {
```

Comment:
thanks for fixing. I just looked at `difference` from the other columns and noticed there's probably another bug here which is that this doesn't check for missing values. what do you think about fixing that too while you're at it?

the other inconsistency is that this method starts with `current = 1` and the others have an `if` check instead. this one is cleaner and it'd also be good to change the others to match the way it's done here for consistency if you're up for it

Reply:
Thanks @benmccann !
Sure, I will update this PR with the changes you suggested tomorrow.